DataFormats/L1THGCal/interface/HGCalMulticluster.h

```diff
@@ -13,7 +13,7 @@ namespace l1t {
     public:
        
       HGCalMulticluster(){
-	this->setClusterStep(ClusterStep::StepMulticluster);
+	this->setClusterStep(ClusterStep::SecondLayer);
```

Comment:
Same as above, it should probably be the algo that decides which `ClusterStep` it is